helpers/auth/error_handler.py

```diff
@@ -14,6 +16,16 @@ def __init__(self, model_obj, entity_name, entity, **kwargs):
     def __enter__(self):
         try:
             self.model_obj.save()
+        except exc.ProgrammingError as error:
+            if (os.getenv('APP_SETTINGS') in ("testing", "development")):
+                long_error_message = str(error.orig)
+                index = long_error_message.find("\n")
```

Comment:
What does this index represent?

Reply:
this counts the characters in the string till I get to the "\n" so when I'm giving the specific error I stop at the exact number of characters till the break.